config.mk

```diff
@@ -5,3 +5,5 @@ PACKAGES_LIST_DEFAULT=default backbone
 OPENWRT_SRC=git://git.openwrt.org/15.05/openwrt.git
 OPENWRT_COMMIT=64e116779c0f7da6d98068b8e7c50f528c8a91f2
```

Comment:
@SvenRoederer you don't have to change this if you want to create a "special" build. let's say you create a new config.mk variable called REPO_PATH and override it via make ARGUMENT when building a release.